### PR TITLE
Add recipe for svglib

### PR DESCRIPTION
**Pull request by nicoddemus**
cc @denivyruck

### CODE REVIEW

**Comment by nicoddemus** [comment]
The AppVeyor build has actually passed but the GH UI apparently missed the notification, so I think this is now ready for merging. 👍

**Comment by nicoddemus** [comment]
@conda-forge/core gently ping. 😁

File: svglib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - pytest-runner
+    - setuptools
+    - reportlab
+    - lxml
```
Comment (jakirkham) [inline svglib/meta.yaml]: Are all of these need at build time or can some of them be dropped from `build` (e.g. `reportlab` and `lxml`)?

File: svglib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - tests
+
+  commands:
+    # see deeplook/svglib#70 why skipping some tests on Windows
+    - pytest tests -k "not test_convert_pdf"  # [win]
+    - pytest tests  # [not win]
+
+about:
+  home: https://github.com/deeplook/svglib
+  license: LGPL
```
Comment (jakirkham) [inline svglib/meta.yaml]: What version of LGPL?

File: svglib/meta.yaml (hunk 1 above)
Comment (nicoddemus) [inline svglib/meta.yaml]: Good point, let me see (this was generated from `conda skeleton` I think). I'm pretty sure we can drop `pytest-runner` as well, will try removing the others locally.

File: svglib/meta.yaml (hunk 2 above)
Comment (nicoddemus) [inline svglib/meta.yaml]: v3, should I add `LGPL v3`? Btw, is there a resource somewhere that lists what should be put here, or is this field considered free form in general and in this case just missing some form of version specifier and `LGPL 3`,  `LGPL v3`, and `LGPL3` would all be acceptable?

File: svglib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{% set name = "svglib" %}
+{% set version = "0.8.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "1f8cb633f154b83987e3a906a55cf69c8739c0df5ea4eee024677c3e8fb62f55" %}
+package:
```
Comment (jakirkham) [inline svglib/meta.yaml]: nit: Please add a newline between the Jinja and `package`.

File: svglib/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline svglib/meta.yaml]: We haven't been too picky. Normally have gone with `LGPL 3` myself.

File: svglib/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline svglib/meta.yaml]: Yeah every time I have seen `pytest-runner` show up, it seems it needs to be in `build` requirements.

**Comment by nicoddemus** [comment]
Applied all requested changes. Thanks @jakirkham for the review!

File: svglib/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  requires:
+    - pytest
+
+  source_files:
+    - tests
+
+  commands:
+    # see deeplook/svglib#70 why skipping some tests on Windows
+    - pytest tests -k "not test_convert_pdf"  # [win]
+    - pytest tests  # [not win]
```
Comment (jakirkham) [inline svglib/meta.yaml]: xref: https://github.com/deeplook/svglib/issues/70

**Comment by jakirkham** [comment]
Thanks @nicoddemus.

**Comment by jakirkham** [comment]
Also as FYI there is a `conda-forge/staged-recipes` team that can be pinged for review. Mainly mentioning this as it is a larger group of people that can review and merge here. So should be more responsive.

**Comment by nicoddemus** [comment]
Thanks, did not know what. :+1:

**Comment by jakirkham** [comment]
Oops, apparently this was in the wrong directory. :man_facepalming: Fixing in PR ( https://github.com/conda-forge/staged-recipes/pull/3054 ).

**Pull request merged**